Source: http://comments.gmane.org/gmane.comp.lang.javascript.nodejs/18924
Upstream: Unknown
Reason: Fix build with -DSHARED_LIBEV=True

diff --git a/cmake/configure.cmake b/cmake/configure.cmake
index c055753..cd058a6 100644
--- a/cmake/configure.cmake
+++ b/cmake/configure.cmake
@@ -90,7 +90,6 @@ add_definitions(
   -DX_STACKSIZE=65536
   -D_LARGEFILE_SOURCE
   -D_FILE_OFFSET_BITS=64
-  -DEV_MULTIPLICITY=0
   -D_FORTIFY_SOURCE=2
   )
 
